daemon/prune.go

```diff
@@ -25,6 +25,27 @@ var (
 	// ErrPruneRunning is returned when a prune request is received while
 	// one is in progress
 	ErrPruneRunning = fmt.Errorf("a prune operation is already running")
+
+	containersAcceptedFilters = map[string]bool{
+		"label":  true,
+		"label!": true,
```

Comment:
@thaJeztah 
- I agree that this is strange, following our discussion, I will open another PR to rewrite this logic.
- I don't think `filters` can contain any space, either.